src/AudioSystem.cpp

```diff
@@ -1,14 +1,27 @@
 #include <iostream>
+#include <utility>
+#include <vector>
 
 #include "AudioSystem.h"
 
 #include "SDL.h"
 #include "SDL_mixer.h"
 
 namespace {
-    const int MIX_FREQ_HZ = 44100; // Sound sampling frequency in Hz. Change to "22050" if too slow.
-    const int MIX_NUM_CHANNELS = 2; // Number of sound channels for output
-    const int MIX_CHUNK_SIZE = 4096; // Bytes used per output sample
+    // Sound sampling frequency in Hz. Change to "22050" if too slow.
+    const int MIX_FREQ_HZ = 44100;
+    // Number of sound channels for output
+    const int MIX_NUM_CHANNELS = 2;
```

Comment:
This is really small, I think it should be bigger.

Reply:
If you mean MIX_NUM_CHANNELS, that just refers to 1 = mono sound, 2 = stereo sound. This doesn't seem to have anything to do with mixing channels. I'll go make that clearer in the comments.

https://www.libsdl.org/projects/SDL_mixer/docs/SDL_mixer_11.html